test/assets/configs/storage.js

```diff
@@ -4,8 +4,9 @@ function getSciServerFilesConfig() {
     const username = getSciServerUsername();
     const password = getSciServerPassword();
     const volume = `${username}/deepforge_test`;
+    const volumePool = 'Storage';
```

Comment:
We should probably change this to "Temporary" now that we are making this possible.